cmake/Platform.cmake

```diff
@@ -18,8 +18,11 @@ if (MSVC)
     # /wd4267
     # Suppress warnings: assignment of 64-bit value to 32-bit QuantLib::Integer (x64)
 
+    # /wd4819
+    # Suppress warnings: The file contains a character that cannot be represented in the current code page
+
```

Comment:
Where was the warning emitted?  I don't see it in the Appveyor build.

Reply:
This warning is emitted on my Windows PC with active code page 949.